src/main/java/entities/Message.java

```diff
@@ -0,0 +1,74 @@
+package entities;
+
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Message {
+    private String body;
+    private String author;
```

Comment:
author should be an object of type User. Adjust attribute and methods accordingly. :)

Reply:
On second thought, we will merge as is and you can change once we have the User class merged into main!